qutip/matplotlib_utilities.py

```diff
@@ -185,3 +185,91 @@ def __init__(self, midpoint=0, vmin=None, vmax=None, clip=False):
     def __call__(self, value, clip=None):
         x, y = [self.vmin, self.midpoint, self.vmax], [0, 0.5, 1]
         return np.ma.masked_array(np.interp(value, x, y))
+
+# Added 16 December 2015
+# by Alexander Pitchford
```

Comment:
Author credits are available in the git history and does not need to be added explicitly in the files (because it doesnt scale when many authors work on a project)